Python Tutorial Exercises(Week 3)/#4.py

```diff
@@ -0,0 +1,32 @@
+
+'''
+This script uses both while loops and for loops to print everyone's full name correctly 
+without using list methods like reverse().  
+This script uses [::-1] to reverse the' last_names' list and 
+uses zip() to merge two lists to one list then print the list out at the end. 
+
+The While loop is pretty much the same. Convert the list first and 
+reverse the list by using len(last_names) - 1, then print both values 
+by using a nested while loop.
+'''
+first_names  = ["Tianchi", "Ariel", "Patrick", "Andrew"]
+last_names = ["Jane", "Savill", "Yap", "Ma"]
+
+#for loop
+for v,b in zip(first_names,last_names[::-1]):
+    print(v,b)
+
+#Wile Loop
+first_names  = ["Tianchi", "Ariel", "Patrick", "Andrew"]
+last_names = ["Jane", "Savill", "Yap", "Ma"]
+
+x = len(last_names) - 1
+y = len(first_names)
+z = 0
+
+while z < y and x >= 0:
```

Comment:
There's a way ti do this without the `x` variable. Have you tried what I had suggested before? Again, try printing the output for these
```
last_names[-1]
last_names[-2]
```